include/wlr/types/wlr_seat.h

```diff
@@ -194,6 +194,9 @@ struct wlr_seat {
 	struct wlr_seat_keyboard_state keyboard_state;
 	struct wlr_seat_touch_state touch_state;
 
+	struct wlr_pointer_constraints_v1 *pointer_constraints;
+	struct wlr_pointer_constraint_v1 *current_pointer_constraint;
```

Comment:
I'm not a fan of this, is there a way to remove these?

Reply:
Now that I think about this, I could make a list inside the pointer constraints manager object, and have the relevant pointer constraints there. Would that be preferred?